### ISSUE TITLE
Code Table Request - REMOVE endoparasites detected

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6224]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

[endoparasites detected](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#endoparasites_detected) should be changed to

* yes-->https://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=detected#detected
or
* no-->https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#not_detected

with a value coming from another CT request, probably shared by https://github.com/ArctosDB/arctos/issues/6222

Data:
[temp_enpd.csv.zip](https://github.com/ArctosDB/arctos/files/11383808/temp_enpd.csv.zip)

Summary:

```
 guid_prefix | count 
-------------+-------
 MVZ:Mamm    |     6
 ASNHC:Mamm  |   165
 DMNS:Bird   |  3804
 UWYMV:Mamm  |     2
 NMU:Mamm    |  1720
 DMNS:Mamm   |  3281
 UAM:Mamm    |     4
 MSB:Mamm    | 13826
 MVZ:Herp    |    30
 CHAS:Mamm   |     1
 MSB:Herp    |   147
 UNCG:Mamm   |   263
 MSB:Host    |  2936
 MLZ:Bird    |     1
 UMZM:Mamm   |     4
 UCM:Bird    |     2
 UMNH:Mamm   |   185
 MSB:Bird    |   110
 UWYMV:Bird  |    18
 UMNH:Herp   |     1
 UMZM:Bird   |     8
 MVZ:Bird    |   817


```

Users:

 @ebraker
 @bryansmclean
 @mkoo
 @campmlc
 @ccicero
 @amgunderson
 @atrox10
 @cjconroy
 @makaylaeasley
 @jtgiermakowski
 @wellerjes
 @keg34
 @AdrienneRaniszewski
 @acdoll
 @jldunnum
 @jrdemboski
 @msbparasites
 @ewommack
 @kderieg322079
 @catherpes
 @adhornsby
 @droberts49

_**Context**: Describe why this new value is necessary and existing values are not._

We have two ways of doing the same thing.

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Code table cleanup should always be highest priority.

_**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by campmlc** [comment]
I support this for MSB collections. However, we need first to add "endoparasites" as an option in
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected

And the detected/not detected attributes will need to be expanded to include the following collections:
Bird
Herp
Inv
Teach

And I assume all existing metadata fields will also transfer over.

**Comment by Jegelewicz** [comment]
As #6232 is complete, can we proceed with this? Do we need specific approval of all affected collections or do we make an announcement about the change, provide a bit of time for feedback, then make a mass change?

Also, can we just do this, or do we need a separate issue?

```
And the detected/not detected attributes will need to be expanded to include the following collections:
Bird
Herp
Inv
Teach
```

**Comment by campmlc** [comment]
On reflection, Id like to create the specific endoparasite values in the code table, eg cestode, nematode, etc and then see how many I can parse into those values first based on remarks metadata.

**Comment by Jegelewicz** [comment]
> see how many I can parse into those values first based on remarks metadata.

I'm happy to help do that once the values are available!

**Comment by acdoll** [comment]
So is this just going to become a table of parasite/virus/bacteria taxonomy?

**Comment by campmlc** [comment]
The attribute will be "examined for" or "not examined for" with a code table dropdown of various values. See currently proposed values here: https://docs.google.com/spreadsheets/d/1bboO2LLBd1D26ykbomQvIFU9ocOH9L9V0lBqCSipaBc/edit#gid=0
Then there will be detected or not detected attributes drawing from the same code table. 
So to convert our current value of "endoparasite examination" and "endoparasites detected", we'd have the following:

endoparasite examination becomes:  "examined for" = endoparasites (or some other more specific value you can choose from the code table, e.g. "helminths" or "cestodes"

endoparasites detected can be more specific: "detected" = "endoparasites" or "helminths" or "cestodes" or "Haemosporida" etc

And we should have the option of "not examined for" and "not detected" for the same code table values.

**Comment by acdoll** [comment]
> See currently proposed values

Ok, that seems reasonable. Sorry, I'm not fully up to speed with these issues. Seeing the Element options in this table has me wondering if this record attribute could also be used as a part attribute option. That probably complicates things further, but it might be a useful option down the road.

**Comment by dustymc** [comment]
>part attribute option

Yea, this has found its way there a bunch of times now....

**Comment by dustymc** [comment]
I believe this was sufficiently approved by @ArctosDB/arctos-code-table-administrators and I would like to begin work on it TOMORROW.

@campmlc @ccicero @jldunnum please confirm and check some boxes.

**Comment by dustymc** [comment]
fresh data:

[Archive 5.zip](https://github.com/ArctosDB/arctos/files/11520839/Archive.5.zip)

done
